src/plugins/vis_type_markdown/public/markdown_vis_controller.test.tsx

```diff
@@ -18,11 +18,14 @@
  */
 
 import React from 'react';
-import { render, mount } from 'enzyme';
+import { wait } from '@testing-library/dom';
+import { render, cleanup } from '@testing-library/react/pure';
```

Comment:
I struggled with making this test work with `enzyme` for quite a while. Ended up with re-imlementing it with simpler `@testing-library`